### PR TITLE
Create a link from the dataexplorer to the navigator.

### PR DESCRIPTION
**Pull request by ConnorStroomberg**
The data explorer setting may be used to hide/show the link
The navigator is opened in the package containing the entity.

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] Test plan template updated
- [x] Clean commits

### CODE REVIEW

File: molgenis-navigator/src/main/frontend/src/store/actions.js [hunk 1]
```diff
@@ -9,9 +9,10 @@ export const QUERY_ENTITIES = '__QUERY_ENTITIES__'
 export const RESET_STATE = '__RESET_STATE__'
 export const GET_STATE_FOR_PACKAGE = '__GET_STATE_FOR_PACKAGE__'
 export const GET_ENTITIES_IN_PACKAGE = '__GET_ENTITIES_IN_PACKAGE__'
+export const GET_ENTITY_PACKAGES = '__GET_ENTITY_PACKAGES__'
 
 /**
- * Resets the entire state using the given packages as the package state.
+ * Resets the entire stat1 using the given packages as the package state.
```
Comment (Mark-de-Haan) [inline molgenis-navigator/src/main/frontend/src/store/actions.js]: typo

**Comment by Mark-de-Haan** [comment]
Missing the test plan

File: molgenis-dataexplorer/src/main/resources/js/dataexplorer.js [hunk 2]
```diff
@@ -426,6 +428,23 @@ $.when($,
         }
 
         /**
+         * Sets the href for the link to the Navigator plugin
+         * The link href to the Navigator is appended with a 'lookup' query that resutls in the navigator opening  in
```
Comment (Mark-de-Haan) [inline molgenis-dataexplorer/src/main/resources/js/dataexplorer.js]: results*

**Comment by Mark-de-Haan** [comment]
screen shot 2017-09-12 at 11 14 43
screen shot 2017-09-12 at 11 13 36
screen shot 2017-09-12 at 11 13 31

And I am getting a very weird error. Every time I open the settings screen in the dataexplorer, every text input field is appended with a ','
screen shot 2017-09-12 at 11 23 30

**Comment by ConnorStroomberg** [comment]
@Mark-de-Haan I can not reproduce the error on my local machine. I think the error may be due to the database not being cleaned before application startup. I suggest we discuss if the database needs to be cleaned before startup up with this feature included or not.

**Comment by ConnorStroomberg** [comment]
@Mark-de-Haan I have updated the [test plan](https://docs.google.com/spreadsheets/d/17KTnwRdwwtTUBuqzUSXUSPFH1IfQB8YovbFptGrRkBU/edit ), and included test steps for the navigator link.

**Pull request merged**